iohub/reader.py

```diff
@@ -270,8 +275,8 @@ def print_info(path: StrOrBytesPath, verbose=False):
                 f"Time points:\t {reader.shape[0]}",
                 f"Channels:\t {reader.shape[1]}",
                 ch_msg,
-                f"(Z, Y, X):\t {reader.shape[2:]}",
-                f"Z step (um):\t {reader.z_step_size}",
+                f"(Z, Y, X) shape:\t {reader.shape[2:]}",
```

Comment:
These description strings are longer than one tab:
```text
=== Summary ===
Format:          ometiff
Positions:       4
Time points:     20
Channels:        3
Channel names:   ['Cy5', 'DAPI', 'FITC']
(Z, Y, X) shape:         (5, 128, 128)
```

Reply:
Fixed in [unify indentation](https://github.com/czbiohub-sf/iohub/pull/173/commits/d03e99ff025712276bad0a915f985ea330fe3c8c).